app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -2,30 +2,68 @@ class DradisDatatable {
   constructor(tableElement) {
     this.$table = $(tableElement);
     this.dataTable = null;
+    this.tableHeaders = Array.from(this.$table[0].querySelectorAll('thead th, thead td'));
     this.init();
   }
 
   init() {
-    var dataTable = this.$table.DataTable({
+    // Remove dropdown option for <th> columns that has data-colvis="false" in colvis button
+    var colvisColumnIndexes = [];
```

Comment:
`colvis` seems to be a name we've chosen, correct? If so, I'd say we could probably choose a more descriptive name that doesn't require the reader to think so hard about what is going on? `column-visible`?

Reply:
Thanks, updated to something more readable.